TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Components/Gyro.java

```diff
@@ -0,0 +1,68 @@
+package org.firstinspires.ftc.teamcode.Components;
+
+import com.qualcomm.hardware.bosch.BNO055IMU;
+import com.qualcomm.hardware.bosch.JustLoggingAccelerationIntegrator;
+import com.qualcomm.robotcore.eventloop.opmode.OpMode;
+
+import org.firstinspires.ftc.robotcore.external.navigation.AngleUnit;
+import org.firstinspires.ftc.robotcore.external.navigation.AxesOrder;
+import org.firstinspires.ftc.robotcore.external.navigation.AxesReference;
+import org.firstinspires.ftc.robotcore.external.navigation.Orientation;
+
+import java.util.Locale;
+
```

Comment:
I would say take out any gyro code for tomorrow, It hasn't been tested, and will probably cause more problems than it can help.

Reply:
As it is not implemented in an OpMode it will not be an issue.